pkg/apis/eventsource/v1alpha1/types.go

```diff
@@ -391,6 +391,9 @@ type SNSEventSource struct {
 	// Metadata holds the user defined metadata which will passed along the event payload.
 	// +optional
 	Metadata map[string]string `json:"metadata,omitempty" protobuf:"bytes,7,rep,name=metadata"`
+	// ValidateSignature is boolean that can be set to true for SNS signature verification
+	// +optional
+	ValidateSignature bool `json:"validateSignature,omitempty" protobuf:"bytes,8,opt,name=validateSignature"`
```

Comment:
protobuf:"varint,8,...."

Reply:
After making change to models, please run `make codegen`.

Reply:
I need help, following the instructions in https://github.com/argoproj/argo-events/blob/master/CONTRIBUTING.md, I tried running make codegen but I get this error **Unable to find 'protoc': exec: "protoc": executable file not found in $PATH** Not sure why I get this

Reply:
`brew install protobuf`

Reply:
@RiyaJohn - Please let me know if you still have any problem to get it running.

Reply:
I am still stuck at running it, am on an ubuntu instance, did setup protobuf following https://gist.github.com/ryujaehun/991f5f1e8c1485dea72646877707f497. But I get 
```
protoc-gen-gogo: program not found or is not executable
--gogo_out: protoc-gen-gogo: Plugin failed with status code 1.
2020/11/06 10:07:42 protoc -I . -I /tmp/tmp.3BBsEsXodp/src -I ./vendor --gogo_out=/tmp/tmp.3BBsEsXodp/src /tmp/tmp.3BBsEsXodp/src/github.com/argoproj/argo-events/vendor/k8s.io/apimachinery/pkg/util/intstr/generated.proto
2020/11/06 10:07:42 Unable to generate protoc on k8s.io.apimachinery.pkg.util.intstr: exit status 1
```
When I tried googling this error **protoc-gen-gogo: program not found or is not executable**, and setting PATH to have GOROOT, GOPATH, GOBIN. I started getting not found error
` ./hack/generate-proto.sh: line 21: /tmp/tmp.w8Fmsc5lIh/bin/go-to-protobuf: No such file or directory`
So not sure what to do

Reply:
Sorry, this is my bad. I'm working on another PR to get the codegen scripts better.

I created a PR on your repo, can you merge it? https://github.com/RiyaJohn/argo-events/pull/1

Reply:
Merged and accepted new changes to resolve conflicts with master. But I see DCO fail, should I be rebasing and pushing again?

Reply:
Ignore the DCO for now, next time use `git commit -s -m 'xxx'`.

There's only one thing left, change to `protobuf:"varint,8,...."` for `ValidateSignature`

Reply:
Committed the change.